transcom/mymove
# B 19938 int

## [B-19938](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19938)

## Summary

This merge simply adds a new text column to the MTO_Shipments database, titled "diversion_reason" to store the reasons for an MTO Shipment diversion.

## Verification Steps for the Author

These are to be checked by the author.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. After checking out the branch, run a fresh migration to add the column to the table. Verify the column appears with it's description.

#### Any new migrations/schema changes:


## Screenshots
